<launch>

    <node pkg='turtlesim' type='turtlesim_node' name='simulator'/>
    <node pkg='turtlesim' type='turtle_teleop_key' name='teleop' output='screen'/>
    <node pkg='tf2_harjoitus' type='turtle2_stamped_node.py' name='stamped_node'/>

    
    <node pkg='tf2_harjoitus' type='turtle_tf2_broadcaster.py' name='turtle1_broadcaster' output='screen'>]
        <param name='turtle' type='string' value='turtle1'/>
    </node>

    <node pkg='tf2_harjoitus' type='turtle_tf2_broadcaster.py' name='turtle2_broadcaster' output='screen'>
        <param name='turtle' type='string' value='turtle2'/>
    </node>  

    <node pkg='tf2_harjoitus' type='turtle_tf2_broadcaster.py' name='turtle3_broadcaster' output='screen'>
        <param name='turtle' type='string' value='turtle3'/>
    </node>

    <node pkg='tf2_harjoitus' type='turtle_tf2_listener.py' name='turtle1_listener' output='screen'/>
    <node pkg='tf2_harjoitus' type='turtle2_tf2_listener.py' name='turtle2_listener' output='screen'/>  


</launch>